src/pacer.js

```diff
@@ -443,6 +449,32 @@ let PACER = {
     banners.forEach(banner => { banner.remove() });
   },
   
+  // returns document data as an object
+  parsePdfDataFromReceipt: () => {
+    // this method uses regex expressions to match that the case number, document number and the attachment 
+    // number from the receipt table and returns an object with the following attributes:
+    //  - case_num
+    //  - doc_num
+    //  - att_num
+    
+    let image_string = $('td:contains(Image)').text();
+    let regex = /(\d+)-(\d+)/;
+    let matches = regex.exec(image_string);
+    if (!matches) {
+      return null;
+    }
+    let r = {};
+    [ , r.doc_num, r.att_num] = matches
+    r.case_num = $.trim($('tr:contains(Case Number) td:nth(1)').text());
+
+    return r;
+  },
+
+  // returns HTML to create a full page iframe that loads the url passed as an argument
+  getFullPageIframe: (url) => {
+  return `<style>body { margin: 0; } iframe { border: none; }</style>
+                  <iframe src="${url}" width="100%" height="100%"></iframe>`;
+  },
```

Comment:
Weird that this is different from appellate, isn't it? Could we merge these two approaches?

Reply:
If not, we should rename this one as I said in my other comment so it's more like: `make...`

Reply:
They are different because I tried to use this `iframe` in Appellate, but the element didn't cover the entire viewport. 

I reviewed the issue and realized that PDF documents from District Court are wrapped in a simple `HTML page` (the content type of the response is **text/html**), but documents from Appellate are not wrapped ( the content type of the response is **application/pdf**)

The extension keeps the `HTML` elements from the response in District court and inserts the floating button that allows the user to download the file (this is the reason that we still get the whole top menu after the document is loaded), so the `iframe` inserted inherits the **CSS** styles from District court PACER.

For Appellate Court, there's no `HTML` in the response, so the extension inserts the floating button and the iframe in the root of the page, but the style in this `HTML` is not enough to fill the view area, so I created a new CSS class (`full-page-iframe`) and put the iframe inside a div element to fix the issue in Appellate Court.

I think we can use the new approach(put the iframe inside a div with the new class) for district and appellate courts, the iframe should fill the parent's view area.

Reply:
Great. I had a feeling the differences could be merged! Nice work.